yarn.lock

```diff
@@ -2306,6 +2306,89 @@
     read-package-json-fast "^3.0.0"
     which "^3.0.0"
 
+"@parcel/watcher-android-arm64@2.4.1":
```

Comment:
these are needed?

Reply:
These are dependencies of sass. Removed now..